modules/sqlite3/3.50.0/overlay/BUILD.bazel

```
@@ -0,0 +1,32 @@
+package(default_visibility = ["//visibility:public"])
+
+cc_binary(
+    name = "shell",
+    srcs = ["shell.c"],
+    # Disable Bazel's default behavior of linking to `libstdc++` / `libc++`.
+    features = ["-default_link_libs"],
+    deps = [":sqlite3"],
+)
```

Looks like `shell.c` now depends on `libm`:

```
bazel-out/k8-fastbuild/bin/external/sqlite3~/_objs/shell/shell.pic.o:shell.c:function seriesFilter: error: undefined reference to 'ceil'
```

WIP according to upstream: https://www.sqlite.org/forum/forumpost/eb88957695

I think this suggestion will fix it (untested):

```suggestion
cc_binary(
    name = "shell",
    srcs = ["shell.c"],
    # Disable Bazel's default behavior of linking to `libstdc++` / `libc++`.
    features = ["-default_link_libs"],
    linkopts = select({
        "@platforms//os:windows": [],
        "//conditions:default": ["-lm"],
    }),
    deps = [":sqlite3"],
)
```